modules/server_processes_manager/README.md

```diff
@@ -0,0 +1,65 @@
+# Server Processes Manager
+
+## Purpose
+
+The server processes manager module is used to get information about processes that were 
+launched asynchronously by other LORIS modules
+
+## Intended Users
+
+The only user that should have access to this module is the person in charge of
+the administration of the LORIS website
+
+## Scope
+
+This module displays the information relevant to the asynchronous processes
+(active and inactive) launched by other modules. The properties displayed for
+each process are as follows:
+
+- The internal LORIS process ID for the process
+- Pid: the Unix process ID of the command that was launched asynchronously.
+- Type: type of process (only mri_upload is currently implemented).
+- Stdout file: full Unix path of the Output log file on the LORIS server 
+(STDOUT output stream for the process).
+- Stderr file: full Unix path of the Error log file on the LORIS server 
+(STDERR output stream for the process).
+- Exit code file: full Unix path of the file on the LORIS server that will 
+contain the exit code of the process once it finishes.
+- Exit code: Unix exit code for the process. This column will be empty until 
+the process terminates.
+- Userid: the LORIS username of the user that launched the asynchronous process.
+- Start time: time (on the LORIS server) at which the process was started.
+- End time: time (on the LORIS server) at which the process ended. Will be 
+empty until the process finishes.
+- Exit text: summary text describing the process execution result. Will be 
+empty until the process finishes.
+
+By default, all processes (active or not) are displayed on the server processes
+manager page but the page provides a selection filter that can be used to display 
+only specific processes. Filtering can be performed based on the PID, type of 
+process or user ID. Finally, note that upon termination, the temporary files used
+to store the processe's STDOUT, STDERR and exit code are either deleted or left as
+is depending on predefined criteria (e.g. the files associated to processes
```

Comment:
What's the predefined criteria? Is the parenthesis something that module does or something the imaging uploader does?

Reply:
Changed the text so that it is clear that the behaviour described is specific to the imaging uploader.